test/Microsoft.NET.Build.Tests/GivenThatWeWantToVerifyNuGetReferenceCompat.cs

```diff
@@ -180,6 +141,140 @@ public class GivenThatWeWantToVerifyNuGetReferenceCompat : SdkTest, IClassFixtur
             }
         }
 
+        [WindowsOnlyTheoryAttribute]
+        [InlineData("netstandard2.0")]
+        [InlineData("netcoreapp2.0")]
+        public void Net461_is_implicit_for_Netstandard_and_Netcore_20(string targetFramework)
```

Comment:
I wouldn't include the values of the theory parameters in the name of the test.  I'd call it something like `Net461_is_implicit_package_target_fallback`.